File: Lez06_01_protected/Lez06_01_protected/Models/Persona.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Lez06_01_protected.Models
{
    internal class Persona
    {
        public string? Nome { get; set; }
        public string? Cognome { get; set; }
        public string? Titolo { get; protected set; } = "N.D.";
    }
}

File: Lez_TS/lez07_moduli_classi/Persona.ts
export default class Persona{
    nome: string | undefined;
    cognome: string | undefined;

    constructor(nome: string, cognome: string){
        this.nome = nome;
        this.cognome = cognome
    }

    saluta() : void {
        console.log(`Ciao ${this.nome} ${this.cognome}`);
        
    }
}